src/main/java/uk/ac/ic/wlgitbridge/server/Oauth2Filter.java

```diff
@@ -112,4 +112,10 @@ private void unauthorized(HttpServletResponse response) throws IOException {
         unauthorized(response, "Unauthorized");
     }
 
+    private void forbidden(HttpServletRequest request, HttpServletResponse response, String message)
```

Comment:
I think we can do two things here:
* Return a 401 with the same message.
* Return a 403. I'm not sure if a 403 here will cause the git client to ask for credentials again. It's good for usability but we're giving some information (the user does exists).